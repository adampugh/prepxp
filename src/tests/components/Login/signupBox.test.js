import React from "react";
import { shallow } from "enzyme";

import SignupBox from "../../../components/Login/signupBox";

describe("<LoginBox />", () => {
    it("should match snapshot", () => {
        const wrapper = shallow(<SignupBox />);
        expect(wrapper).toMatchSnapshot();
    });
});