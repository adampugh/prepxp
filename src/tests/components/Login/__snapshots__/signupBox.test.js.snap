// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoginBox /> should match snapshot 1`] = `
<div>
  <div
    className="login__box"
  >
    <h2>
      Sign up for an account
    </h2>
    <p
      className="error"
    />
    <div
      className="login__box__input"
    >
      <input
        placeholder="name"
        type="text"
      />
      <i
        className="fas fa-user"
      />
    </div>
    <div
      className="login__box__input"
    >
      <input
        placeholder="email"
        type="email"
      />
      <i
        className="fas fa-envelope"
      />
    </div>
    <div
      className="login__box__input"
    >
      <input
        placeholder="password"
        type="password"
      />
      <i
        className="fas fa-lock"
      />
    </div>
    <button
      className="btn"
      onClick={[Function]}
    >
      Sign Up
    </button>
  </div>
  <div
    className="login__bottomBox"
  >
    <p>
      Already have an account? 
      <span>
        Log in
      </span>
    </p>
  </div>
</div>
`;
